### PR TITLE
[Story] Add detail page but only text, image, and video component that can get rendered

### PR DESCRIPTION
**Pull request by grevicoc**
i think this is not done yet, need:

- use useFetch instead manual axios
- add vistocks
- add conditions so other component can get rendered
- make automatically call api get specific post

why i open this PR is to request review from the others about my code, im sorry it's so hard to just make this program running:(

### CODE REVIEW

File: src/pages/Detailpage/Detailpage.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    // Hide,
+    Box,
+    VStack} from "@chakra-ui/react";
+import { DetailPost } from "../../types/types";
+import { parse } from "node-html-parser";
+import useFetch from "../../hooks/useFetch";
+import Render from "./Render";
+
+const Detailpage: React.FC<DetailpageProps> = (props) => {
+    const response = useFetch<any>("http://localhost:2368/ghost/api/content/posts/62d9674aea5970764c913c7c?key=c11259cec5c2cfa1a037f3f5a4&include=tags");
```
[the remark this answers is not in the document]
Comment (tito-feedloop) [reply src/pages/Detailpage/Detailpage.tsx:15]: ini kemaren bermasalah di data ree, data dari ngefetch sebuah post itu hasilnya 1 object doang sedangkan tipe datanya si data itu T[] jadi merah2 deh.

**Comment by tito-feedloop** [comment]
- [ ] Markdown
- [ ] HTML
- [ ] Gallery
- [ ] Divider
- [ ] Bookmark
- [ ] Email content
- [ ] Email call to action
- [ ] Public preview
- [ ] Button
- [ ] Callout
- [ ] Toggle
- [ ] Audio
- [ ] File
- [ ] Product
- [ ] Header
- [ ] Youtube URL
- [ ] Twitter URL
- [ ] Unsplash URL
- [ ] Vimeo URL
- [ ] CodePen URL
- [ ] Spotify URL
- [ ] SoundCloud URL
- [ ] NFT URL
- [ ] Others URL

Boleh tolong dipilih @IloveNooodles yang prioritas dikerjain yang mana, biar gue ama januar prioritasin itu dulu

File: src/pages/Detailpage/Detailpage.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (IloveNooodles) [reply src/pages/Detailpage/Detailpage.tsx]: hooo okeh okeh nanti ak benerin

File: src/pages/Homepage/Homepage.tsx [hunk 2]
```diff
@@ -2,139 +2,37 @@ import React from 'react';
 import BlogPost from '../../components/BlogPost';
 import BaseLayout from '../../layout/BaseLayout';
 import { Post } from '../../types/types';
-
-export const listOfPosts: Post[] = [
```
Comment (IloveNooodles) [inline src/pages/Homepage/Homepage.tsx:6]: ini comment duluuu aja yakk jangan di delete soalnya ga semua nge run lokalnyaaa

**Review by IloveNooodles — APPROVED** [review]
approved with some comment

File: src/pages/Homepage/Homepage.tsx (hunk 2 above)
Comment (grevicoc) [reply src/pages/Homepage/Homepage.tsx:6]: oh okee

**Pull request merged**